.buildkite/package.yml

```diff
@@ -1,8 +1,8 @@
 notify:
   - slack: "#apm-server"
-    if: build.state == "failed"
+    if: build.state == "failed" && build.branch =~ /^(main|\d+\.\d+|\d+\.x)$$/
   - slack: "#observablt-bots"
-    if: "build.state != 'passed'"
+    if: build.state == "failed" && build.branch =~ /^(main|\d+\.\d+|\d+\.x)$$/
```

Comment:
for reviewer:
- send only slack notifications if any failure for main or any release branches